src/test/java/ioanandroflorea/tests/Tema8/teste/BasicAjaxTests.java

```diff
@@ -0,0 +1,20 @@
+package ioanandroflorea.tests.Tema8.teste;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import ioanandroflorea.tests.Tema8.pageObjects.BasicAjaxConfirmationPage;
+import ioanandroflorea.tests.Tema8.pageObjects.BasicAjaxPage;
+
+public class BasicAjaxTests extends BaseTest {
+
+    @Test(groups = {"mobile","desktop"})
+    public void codeItInTest() {
+        driver.get("https://testpages.herokuapp.com/styled/basic-ajax-test.html");
+        BasicAjaxPage basicAjaxPage = new BasicAjaxPage(driver);
+        basicAjaxPage.selectCategory("Desktop.xml");
```

Comment:
From what I could saw, the category name is "Desktop" and not "Desktop.xml". This is causing the failure for this test. Please make the according changes in order to make the test to pass.